src/parser/priest/shadow/modules/Abilities.tsx

```diff
@@ -360,8 +359,14 @@ class Abilities extends CoreAbilities {
       {
         spell: SPELLS.UNHOLY_NOVA,
         category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
-        cooldown: 100,
+        cooldown: 60,
         enabled: combatant.hasCovenant(COVENANTS.NECROLORD.id),
+        castEfficiency: {
+          suggestion: true,
+          recommendedEfficiency: 0.8,
+          averageIssueEfficiency: 0.7,
+          majorIssueEfficiency: 0.6,
+        },
```

Comment:
i didn't know we still had these and not purely the "recommendedEfficiency" one, but if it works it works!